apache2/persist_dbm.c

```diff
@@ -609,7 +609,11 @@ static int collection_store_ex(int db_option, modsec_rec *msr, apr_table_t *col)
         if(ag_dbm == NULL) {
             //Create the DB
             root_dcfg = msr->dcfg1->root_config;
+#ifdef _WIN32           
+            dbm_filename = apr_pstrcat(root_dcfg->mp, "Global\\", root_dcfg->data_dir, "/", var_name->value, NULL);
```

Comment:
so it will be global\/<dbname>, is this "\" needed?

Reply:
Sure, I will change the comment

Reply:
There is no double quotes in the dbm_filename, the double quotes is just for string.

Reply:
sorry i mean it's Global\/<dbname>